compiler-rt/lib/msan/msan.cpp

```diff
@@ -352,10 +352,44 @@ void __sanitizer::BufferedStackTrace::UnwindImpl(
 
 using namespace __msan;
 
+#define PRINT_FAULTING_INSTRUCTION(instname)                            \
+  if (__msan::flags()->print_faulting_instruction) {                    \
+    Printf("Instruction that failed the shadow check: %s\n", instname); \
+    Printf("\n");                                                       \
+  }
+
+#define CANNOT_PRINT_FAULTING_INSTRUCTION                               \
```

Comment:
I think this name is kind of confusing

Reply:
I renamed CANT_PRINT_FAULTING_INSTRUCTION to WARN_IF_PRINT_FAULTING_INSTRUCTION_REQUESTED